### PR TITLE
Fix nits in Modular Resources

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/extend/modular-resources/_index.md [hunk 1]
```diff
@@ -77,13 +77,13 @@ If you are creating your own modular resource, follow these steps:
...
 
-A configured **module** can make one or more *modular resources* available for configuration.
+A configured *module* can make one or more *modular resources* available for configuration.
 
 {{% /alert %}}
 
 ### Code your module
 
-Code a module in the Go or Python programming languages with [Viam's SDKs](/program/apis/) that does the following:
+Use [Viam's Go or Python language SDKs](/program/apis/) to code a module that does either of the following:
```
Comment (JessamyT) [inline docs/extend/modular-resources/_index.md:86]: Can you actually do both in one module? If so
```suggestion
Use [Viam's Go or Python language SDKs](/program/apis/) to code a module that does either or both of the following:
```

Comment (npentrel) [reply docs/extend/modular-resources/_index.md:86]: I think as is makes sense. You'll probably want to do either or most of the time?

Comment (npentrel) [reply docs/extend/modular-resources/_index.md:86]: Maybe remove the word language? Feels a bit strange

File: docs/extend/modular-resources/_index.md [hunk 2]
```diff
@@ -561,7 +561,8 @@ The `attributes` available vary depending on your implementation.
 
 The following is an example configuration for a base modular resource implementation.
 The configuration adds `acme:demo:mybase` as a modular resource from the module `my_base`.
-The custom model is configured as a component with the name "my-custom-base-1" and can be interfaced with the Viam [base API](/components/base/#api):
+The custom model is configured as a component with the name "my-custom-base-1".
+Commands can be sent to the base according to the Viam [base API](/components/base/#api):
```
Comment (npentrel) [inline docs/extend/modular-resources/_index.md:565]:
```suggestion
You can send commands to the base according to the Viam [base API](/components/base/#api):
```

**Review by npentrel — APPROVED** [review]
Lgtm % nits

File: docs/extend/modular-resources/_index.md (hunk 1 above)
Comment (JessamyT) [reply docs/extend/modular-resources/_index.md:86]: Okay! I was attempting to preserve the original sentence information but I agree it sounds strange :)
```suggestion
Use [Viam's Go or Python SDKs](/program/apis/) to code a module that does either of the following:
```

**Pull request merged**